pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: ably_flutter
-description: A wrapper around Ably's Cocoa and Java client library SDKs, providing iOS and Android support.
-version: 1.2.5
+description: Flutter plugin for Ably, a realtime messaging service that powers synchronized digital experiences in realtime.
+version: 1.2.6
```

Comment:
Changes to `version` should only be made under a release PR. Also, for what its worth, currently released version is at `1.2.8`.

Reply:
https://github.com/ably/ably-flutter/pull/261/commits/8af11e6540ec7b05dfd1ec34349e8d46e4e4cd53

Reply:
https://github.com/ably/ably-flutter/pull/261/commits/8af11e6540ec7b05dfd1ec34349e8d46e4e4cd53